LocalSettings.php

```diff
@@ -393,6 +393,10 @@
 		'torejorgwiki' => true,
 		'unikumwiki' => true,
 	),
+	'wmgVariables' => array(
```

Comment:
Should be wmgUseVariables. Also, please add extloadwiki and allthetropeswiki too (and don't forget to remove https://github.com/miraheze/mw-config/blob/master/LocalSettings.php#L1259 :)).